lambda-notes/src/App.js

```diff
@@ -1,19 +1,18 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
 import './App.css';
+import styled from 'styled-components'
+
+const MainContainer = styled.div`
+  border: 1px solid red;
+  max-width: 900px;
+  width: 100%;
+  margin: 0 auto;
+`
 
 class App extends Component {
   render() {
     return (
-      <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h1 className="App-title">Welcome to React</h1>
-        </header>
-        <p className="App-intro">
-          To get started, edit <code>src/App.js</code> and save to reload.
-        </p>
-      </div>
+      <MainContainer>Hello</MainContainer>
```

Comment:
Getting rid of all the fluff and just using something initially here like a **MainContainer** is nice, and testing with "Hello" 👍